tools/validate-api-against-girder.py

```diff
@@ -1,16 +1,12 @@
 #!/usr/bin/env python3
 import click
-import requests
 
 from dandi.dandiapi import DandiAPIClient
-from dandi.dandiset import APIDandiset
 from dandi.girder import GirderCli
```

Comment:
let's kill this file altogether -- girder is gone gone

Reply:
Should `tools/instantiate-dandisets.py` (which also uses Girder) be deleted as well?

Reply:
sure, why to drag it now